core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2DArrays/Print2DArraysTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.print2DArrays;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Please use the Baeldung code formatter, static imports should come after non static ones:
https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij

Reply:
modified

Reply:
I still do not see evidence that the Baeldung formatter was used.

You can import into IntelliJ as follows:
https://stackoverflow.com/questions/34950767/how-to-import-xml-code-style-into-intellij-idea-15

And then reformat the code:
https://www.jetbrains.com/guide/java/tips/reformat-file/#:~:text=Sometimes%20code%20formatting%20can%20get,L%20(Windows%2FLinux).

Reply:
Done